tests/unit/test_util.py

```diff
@@ -0,0 +1,18 @@
+"""
+Test the utility methods
+"""
+
+import pytest
+from koji_wrapper.util import convert_to_list
+
+
+class TestConvertToList(object):
```

Comment:
I would add a test for a dict and a bool or float.

Reply:
Will this handle this in the same way as the code it tests, so if we decide to enhance the function, I will add these tests as suggested.

Reply:
That works for me.